aws/scripts/sh/batch_pw_load_targets.sh

```
@@ -0,0 +1,212 @@
+#!/bin/bash
+
+# Playwright and Load Tests Batch Script
+# Adds test-e2e, test-visual, test-visual-ui, test-visual-update, and load-tests targets to Makefile
+
+set -e
+
+echo "#### Adding Playwright and load test targets to Makefile"
+
+# Add E2E Test Target
+cat >> Makefile << 'E2E_TARGET'
```

Shellcheck (0.10.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)